agnpy/emission_regions/blob.py

```diff
@@ -130,12 +137,24 @@ def set_delta_D(self, Gamma, theta_s):
         mu_s = np.cos(theta_s.to("rad").value)
         self.delta_D = 1 / (self.Gamma * (1 - self.Beta * mu_s))
 
+    def reset_gamma_e(self, gamma_e_size=None):
+        if gamma_e_size is not None:
+            self.gamma_e_size = gamma_e_size
+        self.gamma_e_min = self._n_e.gamma_min
+        self.gamma_e_max = self._n_e.gamma_max
+
     def set_gamma_e(self, gamma_size, gamma_min=1, gamma_max=1e8):
```

Comment:
with the reset_gamma_[ep] functions introduced, are the set_gamma_[ep] functions still used anywhere in the code?

Reply:
Removed. Also, the reset_gamma_[ep] were not really needed, so i removed them too. 
I'd like to remove the gamma_[ep]_size as well, because they don't describe the blob physical parameters. But it needs more work so I'll get back to it later.